docs/essentials/widget_language.md

```diff
@@ -419,7 +416,8 @@ Text {
 > **Tips**
 >
 > - `TypographyTheme` configures the text style used in the application as a part of `Theme`. Click [`Introduction to Theme[wip]`] to learn more.
-> - **`BuildCtx` provides many contexts for the application of the `widget!`.**(I wasn't able to figure out what this could mean. There is only `theme` and `windowCtx` )
+> - `BuildCtx` will provide additional contexts for the application of
```

Comment:
Sorry @M-Adoo I don't understand the comment. Why would something be missing?

Reply:
I want to add your comment before.
> There is only `theme` and `windowCtx`

Reply:
The original text says that `BuildCtx` provides many contexts ..., but I didn't understand what that means because there is only `theme` and `windowCtx`. That does not look like "many" to me.

Reply:
It's not "many" 😄.  Your comment reminded me to mention what's in `BuildCtx`. Never mind, I'll modify it and merge this pr.